salt/elife-xpub/scripts/setup-database.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+set -ex
+
+MAX_WAIT=20 # time in seconds
+DB_AVAILABLE_COMMAND="nc -z -w ${MAX_WAIT} ${PGHOST} ${PGPORT}"
+DC_COMMAND="docker-compose -f docker-compose.yml -f docker-compose.formula.yml"
+DB_CREATED_COMMAND="psql -c \"SELECT 'public.entities'::regclass\""
+
+SETUP_ARGS="--username={{ pillar.elife_xpub.database.user }} --password={{ pillar.elife_xpub.database.password }} --email={{ pillar.elife_xpub.database.email }}"
+
+# Run the command to ensure the database is started
+$(${DC_COMMAND} start)
+
+# Wait for the database to come up
+if $(${DB_AVAILABLE_COMMAND})
+then
+    if $(${DC_COMMAND} exec -T postgres /bin/bash -c "${DB_CREATED_COMMAND}")
```

Comment:
I think the logic is inverted here: the database needs to be created if this command fails (add a `!`?)